### DISCUSSION TITLE
Mono-Repo Changelog

### DISCUSSION THREAD

**Discussion by r0n22** [pawamoy/git-changelog#85] [Q&A]
My current project is a mono repo which has 3 specific projects which require change logs.  Which are not all released at the same time.

I was wondering if there is any way of filtering based on which scope (Angular changelog).

As well as have a prefix for tags so that proja_1.0.0 does not get used in projb_1.1.0 change log creation?

**Comment by pawamoy** [comment]
> I was wondering if there is any way of filtering based on which scope (Angular changelog).

There no such feature, sorry. You can open a feature request if you want :slightly_smiling_face: 

> As well as have a prefix for tags so that proja_1.0.0 does not get used in projb_1.1.0 change log creation?

Not sure to understand this one :confused: 

Maybe [git-cliff](https://pypi.org/project/git-cliff/) has the features you need.
